### DISCUSSION TITLE
Does Coil have a way to realize the Glide's thumbnail functions

### DISCUSSION THREAD

**Discussion by gkj17** [coil-kt/coil#951] [Q&A]
private void loadImageThumbnailRequest() {  

    DrawableRequestBuilder<String> thumbnailRequest = Glide
        .with( context )
        .load( ** );

    Glide
        .with( context )
        .load( * )
        .thumbnail( thumbnailRequest )
        .into( imageView );
}

Does `Coil` have a way to realize the above `Glide` functions.(thumbnail...)

**Comment by colinrtwhite** [comment]
Not at the moment. The feature is tracked here: https://github.com/coil-kt/coil/issues/27

**Reply by gkj17** [reply]
thanks!
